From 0674a418cd6c0d6dd25c298ab2d650fc134c0a8f Mon Sep 17 00:00:00 2001
From: PyScotch <pyscotch@noreply.invalid>
Date: Tue, 13 Oct 2026 16:15:33 +0000
Subject: [PATCH 204/240] Run the `atst' and `mcv' program tests from the check
 target [report R. Perry]

`check_prog_atst' and `check_prog_mcv' were defined in src/check/Makefile
but were not listed in CHECKSCOTCH, so that `make check' never ran
them, while the CMake test list does.

Refs: BLD-18
---
 src/check/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/check/Makefile b/src/check/Makefile
index 6a42c94..9d12a40 100644
--- a/src/check/Makefile
+++ b/src/check/Makefile
@@ -229,6 +229,7 @@ CHECKSCOTCH = 		check_common_file_compress		\
 			check_prog_amk				\
 			check_prog_amk_grf			\
 			check_prog_acpl				\
+			check_prog_atst				\
 			check_prog_gbase			\
 			check_prog_gmap				\
 			check_prog_gmk				\
@@ -243,6 +244,7 @@ CHECKSCOTCH = 		check_common_file_compress		\
 			check_prog_gpart_remap			\
 			check_prog_gscat			\
 			check_prog_gtst				\
+			check_prog_mcv				\
 			check_prog_mmk				\
 			check_prog_mord
 
-- 
2.47.3

